docs/manuals/source/Configuration/Director.rst

```diff
@@ -713,26 +713,26 @@ The directives within an Options resource may be one of the following:
 
 .. config:option:: dir/fileset/include/options/Signature
 
-   :type: <MD5|SHA1|SHA256|SHA512>
+   :type: <MD5|SHA1|SHA256|SHA512|XXH128>
 
    It is strongly recommend to use signatures for your backups.
    Note, only one type of signature can be computed per file.
 
    You have to find the right balance between speed and security. Todays CPUs have often special instructions that can calculate checksums very fast. So if in doubt, testing the speed of the different signatures in your environment will show what is the fastest algorithm.
-   For example the MD5 message-digest algorithm is a cryptographically broken but, it is still suitable for other non-cryptographic purposes (like calculating a checksum to avoid unintended data change as used by Bareos here) and may be preferred due to lower computational requirements than more recent SHA algorithms.
+   For example the XXH128 algorithm is not cryptographically safe, but it is suitable for non-cryptographic purposes (like calculating a checksum to avoid unintended data change as used by Bareos here) and may be preferred due to lower computational requirements.
```

Comment:
```suggestion
   The XXH128 algorithm is not cryptographically safe, but it is suitable for non-cryptographic purposes (like calculating a checksum to avoid avoid data corruption as used by Bareos here).  Bareos suggests XXH128 as the preferred algorithm due to the fact that it has by magnitudes lower computational requirements. The calculation of the checksum has proven to be the bottleneck in environments with high-speed requirements.
```

Reply:
I agree besides the last sentence.
That can be misinterpreted as "we suggest using XXH128, but we know that it is the bottleneck if you need really high speeds"

Reply:
Maybe then the last sentence needs to be moved to top paragraph, `You have to find the right balance...`